digiwf-integrations/digiwf-okewo-integration/digiwf-okewo-integration-service/src/test/resources/application-itest.yml

```diff
@@ -4,6 +4,103 @@ de:
       digiwf:
         okewo:
           password: 'password'
-          url: 'url'
+          url: 'http://localhost:8089'
           username: 'username'
           benutzerId: 'benutzerId'
+
+
+management:
```

Comment:
application-itest should only overwrite relevant properties. no reason to duplicate the entire file